tests/covalent_tests/file_transfer/strategies/s3_strategy_test.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from furl import furl
+
+from covalent._file_transfer import File
+from covalent._file_transfer.strategies.s3_strategy import S3
+
+
+class TestS3Strategy:
+
+    MOCK_LOCAL_FILEPATH = "/Users/user/data.csv"
+    MOCK_REMOTE_FILEPATH = "s3://covalent-tmp/data.csv"
```

Comment:
does this test require the S3 bucket to exist?

Reply:
No. It is just an example s3 location.